deepcell_toolbox/metrics_test.py

```diff
@@ -459,6 +474,24 @@ def test_calc_iou(self):
 
         assert hasattr(o, 'seg_thresh')
 
+    def test_calc_iou_3D(self):
+        y_true, y_pred = _sample1_3D(10, 10, 30, 30, True, 8)
+        o = metrics.ObjectAccuracy(y_true, y_pred, test=True, is_3d=True)
+
+        o._calc_iou()
+
+        # Check that iou was created
+        assert hasattr(o, 'iou')
```

Comment:
Let's add some tests for asserting errors are thrown with bad data ndim.

Reply:
Done